README.md

```diff
@@ -116,9 +116,12 @@ The following environment variables are supported:
 
    Output directory for target system images and NOOBS bundles.
 
- * `DEPLOY_ZIP` (Default: `1`)
```

Comment:
I wonder if we should preserve backward compatibility and default to `DEPLOY_COMPRESSION=none` if `DEPLOY_ZIP` is '0'. Would be nice not to break things for people who were using the previous variable.

Reply:
I was thinking about this too... It makes sense too.

So I'll add that if `DEPLOY_ZIP=0`, we force `DEPLOY_COMPRESSION=none`, but is there anything I should do for `DEPLOY_ZIP=1` too? I'm thinking about the case were someone wrote it explicitly in their config file (admittedly a rare occurrence, since it was the default).
Also, if both are present (`DEPLOY_ZIP=0` and `DEPLOY_COMPRESSION=somethingotherthannone`)for some reason, which one takes precedence? I'm thinking I could print a warning for the user, and make `DEPLOY_COMPRESSION` take precedence.

Reply:
As you say, `DEPLOY_COMPRESSION` should take precedence and `DEPLOY_ZIP` should be marked as deprecated in the documentation.

> is there anything I should do for `DEPLOY_ZIP=1` too?

probably not

Reply:
OK.
In the README, do you still want an entry for `DEPLOY_ZIP` with the deprecation notice or just a note in the `DEPLOY_COMPRESSION` section?

I print a warning for the user also on the detection of `DEPLOY_ZIP=0`:
```
DEPLOY_ZIP has been deprecated in favor of DEPLOY_COMPRESSION
Similar behavior to DEPLOY_ZIP=0 can be obtained with DEPLOY_COMPRESSION=none
Please update your config file
```

Reply:
> In the README, do you still want an entry for `DEPLOY_ZIP` with the deprecation notice

Yeah, please

Reply:
No worries